Ccq/Console/Command/CommandTesting.php

```diff
@@ -0,0 +1,60 @@
+<?php
+declare(strict_types=1);
+
+namespace RLTSquare\Ccq\Console\Command;
+
+use Magento\Framework\Serialize\Serializer\Json;
+use RLTSquare\Ccq\Publisher\QueuePublisher;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class CommandTesting extends Command
+{
+    private const VAR1 = 'hello';
+    private const VAR2 = 'world';
```

Comment:
Naming convention - since you opted for using constants, it would have been better to go with something like private const INPUT_OPTION_VAR1 = 'var1';. private const INPUT_OPTION_VAR2 = 'var2';. that would emphasize the meaning more clearly.